### PR TITLE
fix for #2560 - Added fees field to loan prepay

### PR DESCRIPTION
**Pull request by aks681**
Added a fees field to loan prepay form.

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

This message was posted by [GitMate.io](https://gitmate.io) with timestamp signature `Thanks for the PR, It will be reviewed Shortly!:1eMvwI:JPBjQ0sosNCNLF2Btll9cWBApL0`

File: app/scripts/controllers/loanAccount/LoanAccountActionsController.js [hunk 1]
```diff
@@ -224,6 +224,7 @@
                     scope.formData.transactionDate =  new Date();
                     resourceFactory.loanTrxnsTemplateResource.get({loanId: scope.accountId, command: 'prepayLoan'}, function (data) {
                         scope.paymentTypes = data.paymentTypeOptions;
+                        console.log(data);
```
Comment (gkrishnan724) [inline app/scripts/controllers/loanAccount/LoanAccountActionsController.js]: remove this line please

Comment (aks681) [reply app/scripts/controllers/loanAccount/LoanAccountActionsController.js]: Oh, sorry. Forgot about that.

**Comment by gkrishnan724** [comment]
@aks681  please squash your commits to one. and then it should be ready to go

**Comment by aks681** [comment]
Ok, done.

**Pull request merged**